index.html

```diff
@@ -180,6 +180,110 @@ <h2>card</h2>
 					In, qui!</ts-card
 				>
 			</article>
+
+			<article>
+				<h2>card</h2>
+				<ts-card no-padding
+					>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias
+					cumque delectus eligendi! Commodi cumque in laboriosam omnis quos
+					tempore! Ad amet at enim necessitatibus, quisquam quos ratione vero.
+					In, qui!</ts-card
+				>
+			</article>
+
+			<article>
+				<h2>file uploader input</h2>
+				<ts-file-uploader-input
+					id="testaki"
+					accepted-file-extensions='["doc", "pdf", "jpg"]'
+					size="medium"
+					rtl
+				></ts-file-uploader-input>
+
+				<script>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The length of this JS script (15) exceeds the maximum set to 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3ALongJavaScriptCheck)